src/Models/Account.js

```diff
@@ -627,6 +633,128 @@ class Account {
             message: 'Your data is being exported, please check your email.'
         };
     }
+
+    /**
+* Creates pin code and sends it to the users' email
+* @method sendPinCode
+* @async
+* @param {number} brandingId - Optional parametar
+* @example
+*     InPlayer.Account.sendPinCode(1234)
+*     .then(data => console.log(data));
+* @return {Object}
+*/
+
+    async sendPinCode(brandingId) {
+        if (!this.isAuthenticated()) {
+            errorResponse(401, {
+                code: 401,
+                message: 'User is not authenticated'
+            });
+        }
+
+        const t = this.getToken();
```

Comment:
Remove this, use `this.Account.getToken().token` directly in the Authorization Header.